daemon/logger/journald/journald.go

```diff
@@ -36,6 +36,26 @@ func init() {
 	}
 }
 
+// sanitizeKeyMode returns the sanitized string so that it could be used in journald.
+// In journald log, there are special requirements for fields.
+// Fields must be composed of uppercase letters, numbers, and underscores, but must
+// not start with an underscore.
```

Comment:
Thanks @vdemeester. The pull request has been updated with a couple of test cases for strings starting with underscore. Let me know if there are any issues.